src/Microsoft.AspNet.Mvc.Xml/MvcOptionsExtensions.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNet.Mvc.Xml;
```

Comment:
wouldnt the existence of this in core cause the package to always be included? Should we consider moving these extensions to the new xml package? Then core can be totally independent of xml??